rclpy/CMakeLists.txt

```diff
@@ -89,6 +89,32 @@ endmacro()
 
 call_for_each_rmw_implementation(target)
 
+
+add_library(
+  "${PROJECT_NAME}_implementation_rmw_agnostic"
+  SHARED src/rclpy/_rclpy_implementation_rmw_agnostic.c
+)
+set_properties("${PROJECT_NAME}_implementation_rmw_agnostic" "")
+if(WIN32)
+  set_properties("${PROJECT_NAME}_implementation_rmw_agnostic" "_DEBUG")
+  set_properties("${PROJECT_NAME}_implementation_rmw_agnostic" "_MINSIZEREL")
+  set_properties("${PROJECT_NAME}_implementation_rmw_agnostic" "_RELEASE")
+  set_properties("${PROJECT_NAME}_implementation_rmw_agnostic" "_RELWITHDEBINFO")
+endif()
+
+target_link_libraries("${PROJECT_NAME}_implementation_rmw_agnostic"
+  ${PythonExtra_LIBRARIES}
+)
+
+target_include_directories("${PROJECT_NAME}_implementation_rmw_agnostic"
+  PUBLIC
+  ${rmw_INCLUDE_DIRS}
+  ${PythonExtra_INCLUDE_DIRS}
+)
+
+install(TARGETS "${PROJECT_NAME}_implementation_rmw_agnostic"
+  DESTINATION "${PYTHON_INSTALL_DIR}/${PROJECT_NAME}")
```

Comment:
Good point, I copy pasted the install rule from line 86 without thinking further. I'll update both